js/components/graph/Edge.js

```diff
@@ -25,7 +24,7 @@ export default class Edge extends React.Component {
       this.setState({ status: "active" });
     }
   }
-
+  
```

Comment:
Please delete the wuitespace on this line.

Reply:
This comment is referring to line 27 right?

Reply:
Yes that's correct; GitHub shows the comment immediately below the line that it's referring to.